packages/swirl-components/src/components/swirl-card/swirl-card.tsx

```diff
@@ -24,6 +25,15 @@ export class SwirlCard {
       "card--interactive": this.interactive || this.href,
     });
 
+    /**
```

Comment:
**issue:** Right now there are weird gaps or missing spacings when omitting one or more slot contents. You could check if a specific slot is filled and set a class accordingly to adjust the styles in these cases.

E.g.
```ts
@Element() el: HTMLElement;

// …

render() {
  // …
  const hasImage = Boolean(this.el.querySelector('[slot="image"]'));
  const hasContent = Boolean(this.el.querySelector('[slot="content"]'));
  const hasHeading = Boolean(this.el.querySelector('[slot="heading"]'));

  const className = classnames("card", {
      "card--elevated": this.elevated,
      "card--interactive": this.interactive || this.href,
      "card--has-image": hasImage,
      // …
    });
```